Products/CMFPlone/browser/jsvariables.py

```diff
@@ -36,6 +37,7 @@ def __call__(self, *args, **kwargs):
 
         props = getToolByName(context, 'portal_properties').site_properties
         portal_url = getToolByName(context, 'portal_url')()
+        base_url = self.request['HTTP_REFERER']
```

Comment:
Can someone please validate this for me, if it is safe to use this request variable? Thank you!